components/project-card/meta/creators.jsx

```diff
@@ -7,18 +7,39 @@ const Creators = (props) => {
 
   let classnames = classNames(`creator d-block open-sans`, props.className);
   let labelText = props.showLabelText ? `Created by ` : ``;
+  let joinedCreators;
 
-  return <p className="my-2"><small className={classnames}>{labelText}{props.creators.join(`, `)}</small></p>;
+  if (typeof props.creators[0] === `string` ) {
```

Comment:
What happens on a mixed list, where you get creators `['Pomax', 1, 25, 332]`?

Reply:
You wouldn't, you'd get either creators from `creators` (array of strings) or `related_creators`(array of objects, which we should be using everywhere from now on). I put this code in before I built in the `showLink` prop, so with that I think I can safely remove this, and simply define the propType of `creators` as an array of objects.